backend/src/main/java/com/google/sps/data/Time.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import java.util.Calendar;
+
+/** Class to convert Time */
+public final class Time {
```

Comment:
(A question from someone who isn't familiar with Java) This class seems to implement some very common functionalities. Don't we have standard Java libraries to do similar things, or can we try to use standard library to avoid writing new functions if possible?

Reply:
The main method (getTimeInMillis()) is pretty specific to our implementation. It takes an input "time" such as 11:00am and converts that to a Date timestamp on the current date but at that time. This implementation isn't final, though, because it all depends on what input we get from the front end input form.

Reply:
Cool. So maybe explain a bit about this special date + time combination in the javadoc? (The name "Time" makes me think it's kind of a standard interpretation of Time)

Reply:
Changed the method name to getDateMillis() to clarify.

Reply:
I agree with Tiancong that "Time" might be too generic a name for this class. 

I'm wondering though if we could leverage the java 8 time libraries to simplify this somewhat. Could we implement similar functionality with:

```
long timestamp = LocalDate.now().atTime(hour, minute).toEpochSecond(ZoneOffset.UTC)
```
https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html